### PR TITLE
fix: troca url moved permanently e conserta regex

### PR DESCRIPTION
**Pull request by luckened**
- conserta a url `"https://www.nationalgeographic.com/photography/photo-of-the-day/"` foi [permanently moved](https://developer.mozilla.org/pt-BR/docs/Web/HTTP/Status/301) pra -> `"https://www.nationalgeographic.com/photo-of-the-day/"`
- faz um ajuste na regex pra pegar corretamente a url da imagem

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ http.get(req, (res) => {
 
   res.on('end', () => {
     try {
-      let result = content.match(/meta property="og:image" content="(.*)"/);
-      result = result[0].match(/content="(.*)"/);
+      let result = content.match(/property="og:image" content="(.*)"/);
```
Comment (gabsprates) [inline index.js]: It is important to get the right content, so we need a `meta` with those properties.

A RegExp that will better fit is `/<meta ([^>]*)property="og:image" content="([^\"]*)"`.

Can you please work based on it?

**Review by gabsprates — COMMENTED** [review]
Thank you so much for updating it!

**Comment by luckened** [comment]
:recycle:

**Review by gabsprates — APPROVED** [review]
Thanks!

**Pull request merged**